resources/definitions/fdmprinter.def.json

```diff
@@ -1399,14 +1399,35 @@
                     "limit_to_extruder": "wall_0_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "z_seam_position":
+                {
+                    "label": "Z Seam Position",
+                    "description": "The position near where to start printing each part in a layer.",
+                    "type": "enum",
+                    "options":
+                    {
+                        "backleft": "Back Left",
+                        "back": "Back",
+                        "backright": "Back Right",
+                        "right": "Right",
+                        "frontright": "Front Right",
+                        "front": "Front",
+                        "frontleft": "Front Left",
+                        "left": "Left"
+                    },
+                    "enabled": "z_seam_type == 'back'",
+                    "default_value": "back",
+                    "limit_to_extruder": "wall_0_extruder_nr",
+                    "settable_per_mesh": true
+                },
                 "z_seam_x":
```

Comment:
We have the convention that CuraEngine slices with only the leaf settings. In this case, Z Seam X and Z Seam Y are the ones that CuraEngine slices with, so the X and Y settings should be child settings of the new Z Seam Position.

This is consistent with other such settings such as Wall Line Count vs. Wall Thickness.

Reply:
Updated with https://github.com/Ultimaker/Cura/pull/5935/commits/a9540c1ff04bd98e1a8e74cefab9f0f0bddf41b0